src/relay/op/tensor/unary.cc

```diff
@@ -270,5 +270,54 @@ RELAY_REGISTER_OP("shape_of")
 .set_support_level(10)
 .set_attr<FTVMCompute>("FTVMCompute", ShapeOfCompute);
 
+
+TVM_REGISTER_NODE_TYPE(SizeAttrs);
+
+bool SizeRel(const Array<Type>& types,
+             int num_inputs,
+             const Attrs& attrs,
+             const TypeReporter& reporter) {
+    CHECK_EQ(num_inputs, 1);
+    auto tt = types[0].as<TensorTypeNode>();
+    CHECK(tt != nullptr);
+    const auto* param = attrs.as<SizeAttrs>();
+    CHECK(param != nullptr);
+    reporter->Assign(types[1], TensorTypeNode::make({1}, param->dtype));
```

Comment:
looks like type of `size` in numpy is scalar?

Reply:
Yeah, both numpy.size and tf.size are scalar

Reply:
but `{1}` here is tensor? I think we need to keep it the same as np

Reply:
good catch, sure I'll update the code